### PR TITLE
simplify packit.yaml: followup to #227

### PR DESCRIPTION
**Pull request by TomasTomecek**
most of the opts use the default value so I dropped them

the build will fail 100% because it requires many different dists:
```
error: Bad file: /builddir/build/SOURCES/setuptools_scm_git_archive-1.1.tar.gz: No such file or directory
error: Bad file: /builddir/build/SOURCES/setuptools_scm-6.0.1.tar.gz: No such file or directory
error: Bad file: /builddir/build/SOURCES/setuptools-56.0.0.tar.gz: No such file or directory
error: Bad file: /builddir/build/SOURCES/packaging-20.9.tar.gz: No such file or directory
error: Bad file: /builddir/build/SOURCES/Cython-0.29.23.tar.gz: No such file or directory
error: Bad file: /builddir/build/SOURCES/build-0.3.1.post1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/coverage-5.5.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pyparsing-2.4.7.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/py-1.10.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pluggy-0.13.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/attrs-20.3.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/iniconfig-1.1.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pytest-xdist-2.3.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pytest-forked-1.3.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pytest-cov-2.12.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pytest-6.2.4.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pyparsing-2.4.7.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pip-21.1.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/pep517-0.10.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/toml-0.10.2.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/setuptools_scm_git_archive-1.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/setuptools_scm-6.0.1.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/setuptools-56.0.0.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/packaging-20.9.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/Cython-0.29.23.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/build-0.3.1.post1.tar.gz: No such file or directory
```

I can see that in GHA you're using an action to fetch them: something similar would need to be done in packit.yaml:
```
actions:
  post-upstream-clone:
  - curl -sO https://...
```

### CODE REVIEW

**Comment by webknjaz** [comment]
> most of the opts use the default value so I dropped them

Yeah, I was keeping them for testing to understand what can be tweaked while playing with the integration.

**Comment by webknjaz** [comment]
@TomasTomecek how to get our upstream tarball built during the SRPM build stage?

File: .packit.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ actions:
   - >-
     bash -c '
     spectool
-    --define "upstream_version ${PACKIT_PROJECT_NAME_VERSION}"
```
Comment (webknjaz) [inline .packit.yml]: Oh, I probably did a bad copy-paste here. This is definitely not what I intended.

Comment (TomasTomecek) [reply .packit.yml]: if packit produced better logs from the SRPM build, you would have figured it out more easily

**Comment by webknjaz** [comment]
I've cherry-picked stuff into #227. Closing this one.

**Pull request closed without merging**